packages/api/src/metrics.js

```diff
@@ -54,22 +66,14 @@ export async function metricsGet (request, env, ctx) {
     '# HELP web3storage_pins_total Total number of pins on the IPFS Cluster',
     '# TYPE web3storage_pins_total counter',
     `web3storage_pins_total ${pinsTotal}`,
+    ...pinsMetrics.map(
+      ({ status, total }) =>
+        `web3storage_pins_total{status="${status}"} ${total || 0}`
+    ),
 
-    '# HELP web3storage_pins_status_queued_total Total number of pins that are queued.',
-    '# TYPE web3storage_pins_status_queued_total counter',
-    `web3storage_pins_status_queued_total ${pinsQueuedTotal}`,
```

Comment:
I would like to change this. But we need to take into account older values. Should we rename things here to Prometheus best practises and do a sum in Grafana query with the older name?

cc @hugomrdias @alanshaw

Reply:
yes!